packages/rum/test/e2e/standalone-html/index-async.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>test</title>
+  <meta charset="UTF-8" />
+  <link rel="icon" href="data:;base64,iVBORw0KGgo=" />
+  <script type="application/javascript">
+    /**
+     * Used for testing errors on unsupported browsers
+     */
+    window.capturedTestErrors = []
```

Comment:
this is only used for failsafe browsers.. we don't need it here unless we want to test the async version on failsafe tests as well.